README.md

```diff
@@ -14,20 +14,14 @@ Auro's Design System web component generator is a project tool intended to assis
 [![License](https://img.shields.io/npm/l/@aurodesignsystem/wc-generator.svg?color=blue&style=for-the-badge)](https://www.apache.org/licenses/LICENSE-2.0)
 [![issues](https://img.shields.io/github/issues-raw/AlaskaAirlines/WC-Generator?style=for-the-badge)](https://github.com/AlaskaAirlines/WC-Generator/issues)
 
-It is recommended that the package be installed globally as to ensure quick and easy access to initializing new projects.
-
-```shell
-$ npm i @aurodesignsystem/wc-generator -g
-```
-
-The Auro custom element generator is configured to ensure that you have the latest version of the generator and all it's dependencies prior to starting a new build project.
+It is recommended that the package be executed via a `npx` command.
```

Comment:
I don't think we should say it's recommended. I think we should say this is the only supported method of running the generator.

Reply:
That sentence is really just not needed. Deleted it.